global_body_planner/include/global_body_planner/global_body_planner.h

```diff
@@ -119,6 +140,9 @@ class GlobalBodyPlanner {
     /// Publisher for discrete states in body plan messages
     ros::Publisher discrete_body_plan_pub_;
 
+    /// Publisher for the planning tree
+    ros::Publisher tree_pub_;
+
     /// Topic name for terrain map (needed to ensure data has been received)
     std::string terrain_map_topic_;
```

Comment:
Maybe we can define a typedef for all the frames rather than using std::string everywhere?